projects/baby_driver/scripts/repl_setup.py

```diff
@@ -9,8 +9,11 @@ def setup_default_channel():
 
     # We import these here so that they aren't imported for the REPL
     # pylint: disable=import-outside-toplevel
+    # pylint: disable=unused-import
     import argparse
     import can_util
+    import gpio_port
+    import gpio_set
```

Comment:
Ah, when I wrote import them in repl_setup.py, I meant import them on the top level so that the REPL has access to them. If you type `make babydriver`, you should be able to use `gpio_set`. Also, could you instead do `from gpio_set import gpio_set` and `from gpio_port import GpioPort` so that you can do `gpio_set(...)` from the REPL instead of `gpio_set.gpio_set(...)`?